lib/page/set-errors/set-errors.js

```diff
@@ -91,41 +91,50 @@ const getInstanceErrorStrings = (controlInstance = {}, errorType) => {
 /*
 * @private
 */
-external.getFormattedError = (location, controlInstance, errorType, error, lang) => {
+external.getFormattedError = (position, controlInstance, errorType, error, lang) => {
```

Comment:
I'm curious to know why the param name change here

Reply:
It just seemed a better description, that's all